### PR TITLE
 classification_report for evaluation

### PR DESCRIPTION
**Pull request by bastianentrup**
Using sklearn.metrics classification_report instead of counting and calculating the accuracy directly in the code (also gives more measures and more information on the single classes' evaulations)

### CODE REVIEW

**Comment by ammardodin** [comment]
@bastianentrup Thank you ! Looks good !

**Comment by bastianentrup** [comment]
Thank you for providing the code in the first place!
